spyder/workers/updates.py

```diff
@@ -12,6 +12,8 @@
 import traceback
 
 # Third party imports
+from packaging.version import parse
+import platform
```

Comment:
`platform` is a standard library import, so please move this import to the section above.